frappe/core/doctype/system_settings/system_settings.json

```diff
@@ -117,6 +117,14 @@
    "options": "yyyy-mm-dd\ndd-mm-yyyy\ndd/mm/yyyy\ndd.mm.yyyy\nmm/dd/yyyy\nmm-dd-yyyy",
    "reqd": 1
   },
+  {
+   "default": "HH:mm:ss",
+   "fieldname": "time_format",
```

Comment:
This field is shown in the last section. It should go below **Date Format** field.

![image](https://user-images.githubusercontent.com/9355208/68081815-c13b7100-fe39-11e9-8fc2-ca76b8a6afc3.png)

Reply:
Will be fixed (due to new improved JSON format)